modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,293 @@
+// Module included in the following assemblies:
+//
+// * machine_management/
+// * machine_management/
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-node-custom-partition_{context}"]
+= Optional: Adding a new {op-system} worker node with a custom `/var` partition in AWS
```

Comment:
Should modules generally be marked as "Optional?" In theory, shouldn't I be able to use this module in an assembly where the procedure is mandatory?

Reply:
@maxwelldb I took that from [another module in the Post Install/Node Tasks assembly](https://deploy-preview-40644--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/node-tasks.html#rhel-removing-rhcos_post-install-node-tasks). I'm 100% sure I like having `Optional:` in the heading. But I wanted to keep the parallelism. 
If you believe it should go, I will happily remove it.

Reply:
I believe it should go based on the spirit of mod docs, but I can't point to anything that says it is unacceptable. /shrug